.hlint.yaml

```diff
@@ -53,7 +53,7 @@
 # Ignore some builtin hints
```

Comment:
How often is this file likely to change? Its basiclay (mostly?) autogenerated,.If it changes all the time the it maybe should not be stored in git.

Reply:
I've renamed `.hlint.yaml` to `.hlint.template.yaml` and added `.hlint.yaml` to `.gitignore`.  The script will now generate a `.hlint.yaml` using `.hlint.template.yaml` as the template.